internal/uvm/virtual_device.go

```diff
@@ -31,6 +37,12 @@ type VPCIDevice struct {
 	refCount uint32
 }
 
+// A vpci bus's instance ID is in the form: "VMBUS\vmbusChannelGUIDFormatted\{vmBusInstanceGUID}"
+func (uvm *UtilityVM) GetAssignedDeviceParentID(vmBusInstanceGUID string) string {
+	formattedInstanceGUID := fmt.Sprintf("{%s}", vmBusInstanceGUID)
+	return filepath.Join(assignedDeviceEnumerator, vmbusChannelGUIDFormatted, formattedInstanceGUID)
```

Comment:
I don't think we should use `filepath` here, as these are Windows device paths, not file paths.